Automattic/wp-calypso
# Pattern Assembler - Remove feature flags in preparation to add categories

Related to https://github.com/Automattic/wp-calypso/pull/72144 https://github.com/Automattic/wp-calypso/pull/74060

## Proposed Changes

In preparation for adding the pattern categories, let's remove the flags that are or will be enabled. 

* Remove pattern-assembler/write-flow 
* Remove pattern-assembler/sidebar-revamp

## Testing Instructions



* Click on the Calypso live link on the first comment
* Create a new site
* On the goals screen, choose `Write and publish`
* Continue until the Design Picker and scroll down to click `Start designing`
* Verify that the new sidebar appears and works as expected. See pdtkmj-1eO-p2 
* Click `Create your homepage` and `[ + ]` Add patterns
* Check that posts patterns are in the first position
* Go back to the goals screen and don't choose any goal
* Verify that the posts patterns aren't in the first position 
* Continue to verify that patterns are added to a Home template on your site

## Pre-merge Checklist
